.gitlab-ci.yml

```diff
@@ -135,6 +135,11 @@ variables:
   BCC_VERSION: v0.12.0
   SYSTEM_PROBE_GO_VERSION: 1.14.12
   DATADOG_AGENT_EMBEDDED_PATH: /opt/datadog-agent/embedded
+  # the deb signing key is split in halves, as it exceeds the maximum size of 4096 chars for a single entry
+  DEB_GPG_KEY_NAME: "Datadog, Inc <package@datadoghq.com>"
+  DEB_GPG_KEY_SSM_NAME_PART1: ci.datadog-agent.deb_signing_private_key_8387EEAF_part1
+  DEB_GPG_KEY_SSM_NAME_PART2: ci.datadog-agent.deb_signing_private_key_8387EEAF_part2
```

Comment:
I wonder why we need to have it in two parts here but not in other places we use the key 🤔

Reply:
It depends on whether standard-size secrets or advanced-size secrets are used (https://docs.aws.amazon.com/systems-manager/latest/userguide/parameter-store-advanced-parameters.html).

For instance, the key used to sign the staging repos is stored in standard-size secrets, and thus is also split in two parts: https://github.com/DataDog/datadog-agent/blob/6275ef2adaf6dcacff9a3b0630d2e8ecf7762629/.gitlab/deploy_7/nix.yml#L36-L39